onebusaway-android/src/main/res/values/styles.xml

```diff
@@ -196,4 +196,19 @@
         <item name="android:paddingLeft">5dp</item>
         <item name="android:paddingRight">5dp</item>
     </style>
+
+    <!-- Trip details -->
+    <style name="TripDetailsAgency">
+        <item name="android:textSize">12sp</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textColor">@android:color/white</item>
```

Comment:
Can you please change these colors to `@color/header_text_color`?

The app can be white-labeled, in which case the theme (i.e., Action Bar) colors could change.  If this happens, we want to be able to change the text color of the Action Bar items in one place.  `@color/header_text_color` is currently `@android:color/white`, so this change won't have any impact on the current presentation.